File: Atividades/Reforço de Lógica/Python/EX09.py
# ------------------- Soma de Valores em Lista ------------------- #

lista = [] # Declarando uma lista vazia
usuario = -1 # Declarando a variável que armazena o comando do usuário

while usuario != 0: # Enquanto o usuário passar números diferentes de zero, segue com um loop
    usuario = int(input("Insira um valor ou 0 para sair: ")) # Pedindo um número ao usuário
    lista.append(usuario) # Adicionando esse número à lista

soma = sum(lista) # Somando todos os valores presentes na lista
print(lista) # Printando a lista

print(f"A soma desses números resulta em: {soma}") # Printando o resultado da soma da lista

File: Atividades/Reforço de Lógica/PHP/EX09.php
<?php 
// Declarando as variáveis que usaremos:
$lista = [];
$usuario = -1;

// Loop para registrar os números fornecidos pelo usuário até que ele digite 0:
while($usuario != 0){
    $usuario = readline("Insira um valor ou 0 para sair: \n");
    array_push($lista,$usuario);
}

// Somar a lista:
$soma = array_sum($lista);
// Mostrando a lista:
print_r($lista);
// Mostrando a soma da lista:
echo "A soma desses números resulta em: $soma";
?>